src/object/rigid_body.rs

```diff
@@ -244,7 +247,7 @@ impl<N: Scalar> RigidBody<N> {
     }
 
     /// Creates a new rigid body that can move.
-    pub fn new_dynamic<G>(shape: G, density: N, restitution: N, friction: N) -> RigidBody<N>
+    pub fn new_dynamic<G>(shape: G, density: N, restitution: N, friction: N, groups: Option<RigidBodyCollisionGroups>) -> RigidBody<N>
```

Comment:
I'd prefer not to add groups as a constructor argument. For many applications, a custom value is not needed at all.